rootfs-builder/README.md

```diff
@@ -0,0 +1,57 @@
+# Building a rootfs for Kata Containers Guest OS #
+
+Kata Containers guest OS is created using `rootfs.sh`.
+
+## Supported base OS ##
+
+The `rootfs.sh` script support different based OS `rootfs`. To build
+a `rootfs` based in a supported based OS run:
+
+``` ./rootfs.sh <distro> ```
+
+To check the supported `rootfs` based OS use:
+
+```
+./rootfs.sh -h
+```
+
+## Adding support for new base OS ##
+
+The script `rootfs.sh` will look for directories that meets the following
+structure:
+
+- A bash script that called `rootfs_lib.sh`
+
+This file must contain a function called `build_rootfs()` this function
+must receive as first argument the path where the `rootfs` will be
+populated.
+
+
+- A bash file `config.sh`
+
+This represents the specific configuration for each based OS. It must
```

Comment:
- Indent required.
- Can you reword this a little? I'm confused by "for each based OS".